scout/commands/load/research.py

```diff
@@ -50,55 +66,45 @@ def research(case_id, institute, force):
     default_threshold = 8
     files = False
     for case_obj in case_objs:
-        if force or case_obj["research_requested"]:
-            # Test to upload research snvs
-            if case_obj["vcf_files"].get("vcf_snv_research"):
-                files = True
-                adapter.delete_variants(
-                    case_id=case_obj["_id"], variant_type="research", category="snv"
-                )
-
-                LOG.info("Load research SNV for: %s", case_obj["_id"])
-                adapter.load_variants(
-                    case_obj=case_obj,
-                    variant_type="research",
-                    category="snv",
-                    rank_threshold=default_threshold,
-                )
+        if not (force or case_obj["research_requested"]):
+            LOG.warning("research not requested, use '--force'")
+            continue
+        # Test to upload research snvs
```

Comment:
You now have neat docstrings for the single called function - time to remove the comments!

Reply:
Fixed that. I refactored another small thing (could not keep my fingers away from my old sorrows) so you might want to have a quick look again.